microsetta_interface/templates/source.jinja2

```diff
@@ -140,25 +140,27 @@
     {% if samples|length > 0 %}
     Samples (click on a barcode to provide collection information):
     <div class="list-group">
-        <div class="row">
-            <div class="col-sm">
-                <i>Barcode</i>
-            </div>
-            <div class="col-sm">
-                <i>Collection date</i>
-            </div>
-            {% if is_human %}
-            <div class="col-sm">
-                <i>Sample type</i>
-            </div>
-            <div class="col-sm">
-                <i>Results</i>
-            </div>
-            <div class="col-sm">
-                <i>Sample-specific survey</i>
+        <div class="container">
```

Comment:
Is this paired with a closing `</div>`?

Reply:
Yes, line 143 maps to line 164.  Github's diff isn't super clean for showing tabbing in the inner section.